google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelector.java

```diff
@@ -391,6 +393,25 @@ public void showPopup(RelativePoint showTarget) {
     }
   }
 
+  @Override
+  public void addFocusListener() {
+    super.addFocusListener();
+
+    getTextField().addFocusListener(new FocusListener() {
+      @Override
+      public void focusGained(FocusEvent e) {
+        // do nothing
+      }
+
+      @Override
+      public void focusLost(FocusEvent event) {
+        if (!event.isTemporary()) {
+          onSelectionChanged(getCurrentModelItem());
```

Comment:
This means that `onSelectionChanged` could be called with a null argument. Did you verify that this won't break anything if that happens?